src/SamplesApp/SamplesApp.Droid/MainActivity.cs

```diff
@@ -9,7 +9,7 @@ namespace SamplesApp.Droid
 {
 	[Activity(
 			MainLauncher = true,
-			ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize,
+			ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize | ConfigChanges.UiMode,
```

Comment:
Should this be changed in the solution template? Should also be documented somewhere for existing apps wanting to implement dark mode.

Reply:
That is true. I will update the template and write a doc for it

Reply:
Done